frontend/app/src/locales/en.json

```diff
@@ -2215,6 +2215,7 @@
     "message": "rotki accounts are encrypted using your password and saved in your local filesystem."
   },
   "app": {
+    "help": "Click for help",
```

Comment:
```suggestion
    "help": "Read the documentation",
```

Reply:
Maybe just keep it more simple as `Help`

Reply:
I just am not sure about the help world alone. Made me think of some kind of support forum. But sure if you want just put help.

Reply:
I'm not super crazy about putting this on the right-hand side, and for sure not in the right-most (most important/prominent) position. I also think that if something goes on the bar it should keep the same size/look/feel as the rest.

How would it look/feel if we put it on the left (after the "save status" icon?)? We could eventually maybe flesh out that area and allow a user to pick a few "shortcuts" or quick-links that they could define in their settings (i.e. open specific pages+sub-pages).

Reply:
Contextually it doesn't make sense to add it there since the rest are status icons